simulator/README.md

```diff
@@ -0,0 +1,161 @@
+# trace-to-events
+
+## Introduction
+
+## Command Line Interface
+
+```shell
+simulator [OPTIONS] --broker <BROKER> [COMMAND]
+```
+
+For instance:
+
+```shell
+simulator --broker localhost:19092 \
+    --trace-topic Traces \
+    --event-topic SimulatedEvents \
+    json \
+      --path "trace.json" \
+      --repeat=10
+```
+
+For instructions run:
+
+```shell
+simulator --help
+```
+
+### Commands
+
+- `Single`:            Produce a single trace.
+- `Continuous`:        Produce a regular infinite sequence of traces are regular intervals.
+- `Json`:              Produce traces in the manner specified by a json file.
+
+## Json Format
```

Comment:
Can we call this something other than the format the data is defined in, e.g. "arbitrary", "defined", or similar.